pkg/collector/corechecks/servicediscovery/apm/detect.go

```diff
@@ -37,7 +38,22 @@ const (
 	Injected Instrumentation = "injected"
 )
 
-type detector func(pid int, args []string, envs map[string]string, contextMap usm.DetectorContextMap) Instrumentation
+const (
```

Comment:
Why has this code been moved?

Reply:
I've restored original code. Golang style expects all constants in the beginning of file.